docker/docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       - "9999:9999"
       - "9998:9998"
       - "9997:9997"
+      - "9997:9997/udp"
```

Comment:
This is exposed because the loginserver need to connect the gameserver using udp IP:Port
and IP is now the external one, so the gameserver need to expose it.